setup.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+
+from setuptools import find_packages
+from setuptools import setup
+
+
+# Package meta-data
+NAME = "hepdata-pyhf-extractor"
+INFO = "Python package to summarize pyhf submissions"
+URL = "https://github.com/HEPData/hepdata-pyhf-extractor"
+REQUIRES_PYTHON = ">=3.6, <4"
+AUTHORS = "HEPData and pyhf teams"
+VERSION = open("VERSION", "r").read().strip()
```

Comment:
Any interest in moving everything that doesn't need logic into `setup.cfg`? Reduces everything down pretty nicely to where you can get away with just using

```
from setuptools import setup
```

in `setup.py`.

Reply:
I am not seeing how `setup.py` is difficult to follow as it is right now. Sure, if it grows we can split it into 2 files, but I don't see any benefit with its current state.